sympy/physics/mechanics/tests/test_body.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from sympy import Symbol
+from sympy.physics.vector import Point, ReferenceFrame
+from sympy.physics.mechanics import inertia, Body
+
+
+# To test if two frames were created the same way.
+def check_reference_frames(frame1, frame2):
+    assert frame1.name == frame2.name
+    assert frame1.str_vecs == frame2.str_vecs
+    assert frame1.indices == frame2.indices
+    assert frame1.varlist == frame2.varlist
+    assert frame1._dlist == frame2._dlist
+    assert frame1._ang_vel_dict == frame2._ang_vel_dict
+    assert frame1._ang_acc_dict == frame2._ang_acc_dict
+    assert frame1._dcm_dict == frame2._dcm_dict
+
+
+def test_default():
+    body = Body('body')
+    assert body._name == 'body'
+    assert body.parent is None
+    assert body.child is None
```

Comment:
I though we agreed that the Joint classes would track the parent/child relationship of bodies?